### PR TITLE
In operator from righthand side for SortedRange.

### PR DESCRIPTION
**Pull request by dukc**
For consistency with #5629

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -9856,6 +9856,7 @@ evaluations of $(D pred). See also STL's $(HTTP
 sgi.com/tech/stl/binary_search.html, binary_search).
  */
 
+
```
Comment (MetaLang) [inline std/range/package.d]: Extraneous whitespace

File: std/range/package.d [hunk 2]
```diff
@@ -9865,6 +9866,15 @@ sgi.com/tech/stl/binary_search.html, binary_search).
         return !predFun(value, _input[i]);
     }
 
+/**
+Performs $(D contains) with reverse order of arguments.
```
Comment (MetaLang) [inline std/range/package.d]: This is just stating implementation details. Also needs a `Params:` and `Returns:` section.

Comment (dukc) [inline std/range/package.d]: I object to that latter part, because other member functions lack them too.

**Comment by nordlow** [comment]
I recall somebody saying that D's `in` operator is reserved for O(1)-algorithms. SortedRange.contains is O(log(n)).

**Comment by ZombineDev** [comment]
@nordlow according to http://dlang.org/phobos/std_container (see at the end) `in` taking `O(log(n))` is absolutely acceptable and expected.

**Pull request merged**